README.md

```diff
@@ -136,6 +136,12 @@ This example just streams the body of the request. The body of the response can
 
 Check out the `examples` folder how your server could look like.
 
+The first parameter (often written as `RequestInterface $request` in examples and tests) 
+of the callback function and a middleware is a `ServerRequest` object based on the 
```

Comment:
I would consider to add a short description here. Currently this status of this PR that you can server-side paramters via the callback function or more relevant the middleware functions.

I suggest to add just a simple example and a link to the PSR-7 chapter of `ServerRequestInterface`